Enhancement/update into slide

munroTom opened the pull request:
Styles are generally more responsive as well
<img width="278" alt="Screenshot 2020-06-13 at 14 53 21" src="https://user-images.githubusercontent.com/29929268/84570571-3f73d480-ad86-11ea-9c4a-4f20ea5031e9.png">
<img width="334" alt="Screenshot 2020-06-13 at 14 53 29" src="https://user-images.githubusercontent.com/29929268/84570573-40a50180-ad86-11ea-8149-bf5eb0713886.png">
<img width="359" alt="Screenshot 2020-06-13 at 14 53 38" src="https://user-images.githubusercontent.com/29929268/84570574-41d62e80-ad86-11ea-8f83-ea58d9a96df2.png">

lizheard1 commented:
@munroTom great! What do you mean by responsive styles? As in they fit to different screen sizes better? A few small comments:

- 1st slide: last sentence should be 'map' not 'maps'
- 2nd slide: can wording be moved down slightly so a bit more of a gap between 'Planet Patrol' and the wording? 
- Slide 3 - is this orange the brand guidelines orange? - please could the colour be matched to the orange on the 'snap it'

@joe-mcguinness do you have any feedback from a stylistic perspective?

neilfulwiler commented on src/components/common/Logo/Logo.tsx:
```diff
@@ -1,19 +1,225 @@
   const styles = useStyles();
-  return <img className={styles.logo} src={logo} alt={"Planet Patrol"} />;
+  return (
+    <svg
```
can we put this in a separate .svg file?

munroTom replied:
I tried doing that but then it wasn't super easy to override the fill colour, so in line 30 we set the fill dynamically from the `COLOR` options further up, I'm not super familiar with using svgs inline so there may be a nicer way to do it that I'm not aware of!

joe-mcguinness commented:
> @munroTom great! What do you mean by responsive styles? As in they fit to different screen sizes better? A few small comments:
> 
> * 1st slide: last sentence should be 'map' not 'maps'
> * 2nd slide: can wording be moved down slightly so a bit more of a gap between 'Planet Patrol' and the wording?
> * Slide 3 - is this orange the brand guidelines orange? - please could the colour be matched to the orange on the 'snap it'
> 
> @joe-mcguinness do you have any feedback from a stylistic perspective?

Completely agree with your points, plus we should really be using the Brown STD typeface here. I've attached the brand guidelines here for reference.
[Plastic Patrol Brand Guidelines  (2) (1).pdf](https://github.com/PlasticPatrol/PlasticPatrolWeb/files/4813357/Plastic.Patrol.Brand.Guidelines.2.1.pdf)

munroTom commented:
@joe-mcguinness do you have a copy of the typeface asset that can be used in the app? I can't find it online / the figma project you sent

munroTom commented:
>What do you mean by responsive styles? As in they fit to different screen sizes better?

Yep that's it! Hopefully these should look ok on any screen size, whereas before the spacing/ sizes of assets didn't really change leading to a lot more wasted space on larger screens

joe-mcguinness commented:
[Brown STD.zip](https://github.com/PlasticPatrol/PlasticPatrolWeb/files/4842039/Brown.STD.zip)
@munroTom  attached the font files

joe-mcguinness commented:
> @munroTom great! What do you mean by responsive styles? As in they fit to different screen sizes better? A few small comments:
> 
> * 1st slide: last sentence should be 'map' not 'maps'
> * 2nd slide: can wording be moved down slightly so a bit more of a gap between 'Planet Patrol' and the wording?
> * Slide 3 - is this orange the brand guidelines orange? - please could the colour be matched to the orange on the 'snap it'
> 
> @joe-mcguinness do you have any feedback from a stylistic perspective?
I think the spacing on the 2nd and 3rd screenshots could be improved, and the colour from guidelines is here:
<img width="265" alt="Screenshot 2020-06-28 at 10 59 25" src="https://user-images.githubusercontent.com/31798959/85944444-aec4f900-b92e-11ea-8fdb-8bd572f4cbcd.png">

munroTom commented:
Sorry I didn't update the screenshot - I've updated the orange though, will add the typeface in when I get a moment and see what I can do with the spacing, I've added some more margin between the logo and first bit of text on each slide
